MAXQDA-Conversion/Extract.ipynb

```diff
@@ -0,0 +1,202 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#MAXQDA Conversion\n",
+    "<p>This notebook converts MAXQDA annotations into a format that can be used by Vogon.</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "import pandas as pd\n",
+    "import csv\n",
+    "import string\n",
+    "import time\n",
+    "import os\n",
+    "import urllib\n",
+    "from pathlib import Path\n",
+    "# Uncomment the next two lines if you want to see expanded output\n",
+    "#pd.options.display.max_colwidth\n",
+    "#pd.set_option('display.max_colwidth', -1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Paste the file path to file that contains the text that your annotations came from. To copy file paths on mac right click on the file then press the option key and click on Copy (item name) as Pathname. On windows hold shift and then right click and select 'Copy as path'. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_path = ''\n",
+    "f = open(file_path)\n",
+    "text = f.read()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Paste the file path to csv file that contains the annotations that you would like to convert."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "csv_file_path = ''\n",
+    "annotations = pd.read_csv(csv_file_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Paste the file path to the folder that you would like the generated csv to download to"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "save_folder = ''"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Get position of annotations in text (Mat take a while for large files). Will be in downloads folder"
```

Comment:
do you mean "might take a while..."?

Reply:
"Will be in downloads folder" is not a full sentence and also wrong with the latest changes?